src/sidebar.jsx

```diff
@@ -315,7 +313,7 @@ const DropdownWithKebab = ({
             id: "edit-permissions",
             onClick: () => {
                 editPermissions(Dialogs, {
-                    selected: selected[0] || currentDirectory,
+                    selected: selected[0] || directory_name,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-256-20240207-173252-9bf6d491-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)